deutsche-gesellschaft-fur-psychologie.csl

```diff
@@ -279,6 +283,65 @@
       <text variable="locator" prefix=" "/>
     </group>
   </macro>
+  <macro name="issued-citation">
+    <choose>
+      <if variable="issued">
+        <group>
+          <choose>
+            <if variable="original-date">
+              <date variable="original-date">
+                <date-part name="year"/>
+              </date>
+              <choose>
+                <if variable="translator">
+                  <text value="; übers. "/>
+                </if>
+                <else>
+                  <text value="/"/>
```

Comment:
These 2 text values should be delimiters on a group, otherwise they might be printed when data are missing. The choose test should be wrapped around the whole group